test/e2e/specs/specs-calypso/wp-calypso-seo-preview.js

```diff
@@ -20,18 +20,15 @@ import NavBarComponent from '../../lib/components/nav-bar-component.js';
 import SidebarComponent from '../../lib/components/sidebar-component.js';
 
 const mochaTimeOut = config.get( 'mochaTimeoutMS' );
-const startBrowserTimeoutMS = config.get( 'startBrowserTimeoutMS' );
 const screenSize = driverManager.currentScreenSize();
 const host = dataHelper.getJetpackHost();
 
 describe( `[${ host }] SEO Preview page: (${ screenSize }) @parallel`, function () {
 	this.timeout( mochaTimeOut );
 	let driver;
 
-	before( 'Start browser', async function () {
-		this.timeout( startBrowserTimeoutMS );
-		driver = await driverManager.startBrowser();
-		await driverManager.clearCookiesAndDeleteLocalStorage( driver );
+	before( () => {
+		driver = global.__BROWSER__;
```

Comment:
Can we use the `startBrowser()` helper instead? [It will return the global driver if it exists](https://github.com/Automattic/wp-calypso/blob/2e401878be47d3689db3bc53202b2725a8cd4358/test/e2e/lib/driver-manager.js#L122-L124). Unless we can actually store the `driver` as a global Jest variable and drop the before hooks?

Reply:
> Unless we can actually store the driver as a global Jest variable and drop the before hooks?

I'll try that.

Reply:
After playing with a few patterns, the only two ways I found of get rid of the hook are:

- Either change the tests to use `__BROWSER__` instead of `driver` (which looks ugly), or save it in `global.driver` (which sounds too generic and may clash with something else).
- Save it in `this.driver` so it is accessible from hooks and tests.

I went with the second method. It feels a bit less hacky than using `global`. What do you think?

Reply:
The second method is (as I understand) quite similar to how I manage my browser instance in Playwright. So agreed on your account of that option being 'less hacky'.